src/time_evolution/time_evolution.jl

```diff
@@ -149,9 +149,9 @@ function Base.show(io::IO, sol::TimeEvolutionMCSol)
 end
 
 @doc raw"""
-    struct TimeEvolutionSSESol
+    struct TimeEvolutionStochasticSol
 
-A structure storing the results and some information from solving trajectories of the Stochastic Shrodinger equation time evolution.
+A structure storing the results and some information from solving trajectories of the Stochastic Master Equation time evolution.
```

Comment:
ahh
just a minor change here:
```suggestion
A structure storing the results and some information from solving trajectories of the Stochastic time evolution.
```